packages/components/src/snackbar/types.ts

```diff
@@ -28,8 +32,32 @@ type SnackbarOnlyProps = {
 	listRef?: MutableRefObject< HTMLDivElement | null >;
 };
 
-export type SnackbarProps = Omit< NoticeProps, '__unstableHTML' > &
-	SnackbarOnlyProps;
+export type SnackbarProps = Pick<
```

Comment:
This was the part I had the most trouble with. This component uses the types of the `Notice` component, and I discovered that the following types, which are not available in the `Snackbar` component, also apply to the `Snackbar` component.

- `isDismissible`
- `status`
- `actions.className`
- `actions.noDefaultClasses`
- `actions.variant`

I used `Pick` instead of `Omit` so that when new props are added to the `Notice` component in the future, they are not unintentionally exposed to the `Snackbar` component as well.

At the same time, the code editor should also correctly display the description of the `actions` prop.

![image](https://github.com/WordPress/gutenberg/assets/54422211/8844fdcc-fe0d-4167-b5a1-e73114eda894)